src/fragments/lib/datastore/js/getting-started/30_platformIntegration.mdx

```diff
@@ -56,5 +56,40 @@ DataStore.configure({
 });
 ```
 
+</Block>
+<Block name="Expo">
+
+Start with the [Expo](https://docs.expo.dev/):
+
+```bash
+expo init AmplifyDataStoreExpo
+cd AmplifyDataStoreExpo
+npx amplify-app@latest
+npm install @react-native-community/netinfo @react-native-async-storage/async-storage
+```
+
+**Use SQLite for enhanced performance (optional)**
+
+Expo built apps can now use SQLite with DataStore. SQLite offers considerable performance advantages over the default "AsyncStorage" database.
+
+To enable SQLite with DataStore for enhanced local database performance, follow the steps below:
+
+```sh
+expo init AmplifyDataStoreExpo
+cd AmplifyDataStoreExpo
+npm install aws-amplify @aws-amplify/datastore-storage-adapter expo-sqlite aws-amplify-react-native amazon-cognito-identity-js @react-native-community/netinfo @react-native-async-storage/async-storage @react-native-picker/picker
```

Comment:
To build on @Samaritan1011001's point, why are these steps different than the ones above which makes use of `npx amplify-app@latest`? Shouldn't it basically be the same steps only with the addition of `@aws-amplify/datastore-storage-adapter expo-sqlite`?

Reply:
Yes, I copied the steps from existing guide and it is missing the `npx amplify-app@latest`. I'll add that step to both React Native and Expo steps.